horovod/common/parameter_manager.h

```diff
@@ -90,23 +82,33 @@ class ParameterManager {
   //  tensor_names: The names of the tensors that have been processed.
   //  bytes: The number of bytes that were processed per worker.
   //  microseconds: The number of microseconds taken to process the bytes on this worker.
-  void Update(const std::vector<std::string>& tensor_names, int64_t bytes);
+  bool Update(const std::vector<std::string>& tensor_names, int64_t bytes);
```

Comment:
Add comment about the return value indicating whether we should sync or not.

Reply:
Added.